sparkbench/streaming/pom.xml

```diff
@@ -51,22 +51,6 @@
       </activation>
     </profile>
 
-    <profile>
-      <id>spark1.6</id>
-      <dependencies>
-        <dependency>
-          <groupId>org.apache.spark</groupId>
-          <artifactId>spark-streaming-kafka_${scala.binary.version}</artifactId>
-          <version>1.6.0</version>
-        </dependency>
-      </dependencies>
-      <activation>
-        <property>
-          <name>spark</name>
-          <value>1.6</value>
-        </property>
-      </activation>
-    </profile>
 
```

Comment:
If we deprecate the Spark streaming part(cc @carsonwang ), maybe we should indicate that in README?

Reply:
@gczsjdy  we didn't deprecate the Spark streaming part and just didn't set default value when mvn, if mvn Spark streaming part , we should set “-Psparkbench -Dmodules -PStreaming”.

Reply:
Can we compile successfully as for streaming module?

Reply:
Another issue you mentioned, I still think by default not compiling the streaming part is a little weird...